kata-deploy/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM centos/systemd
```

Comment:
centos:centos7 is more commonly used. would be smaller I think.

Reply:
we need systemd to do the restart of services
```
centos                        centos7             49f7960eb7e4        13 days ago         200MB
centos/systemd                latest              5477808193ce        2 months ago        199MB
```

Reply:
oh. right. thanks.